libcxx/test/libcxx/ranges/range.adaptors/range.concat/iterator.valueless_by_exception.pass.cpp

```diff
@@ -0,0 +1,263 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: std-at-least-c++26
```

Comment:
It's unfortunate that UNIX headers seem necessary.
```suggestion
// REQUIRES: has-unix-headers
// REQUIRES: std-at-least-c++26
```

I guess it's possible to check assertion failures on Windows like MSVC STL's death tests, but that should be done in another PR.